proxyserver/src/main/java/com/lyft/data/proxyserver/ProxyServerConfiguration.java

```diff
@@ -7,13 +7,21 @@
   private String name;
   private int localPort;
   private String proxyTo;
+  private String externalUrl;
```

Comment:
if the purpose it only to override this in ui, no need to add it here.

Reply:
Don't we need it here so that it can be set to the same value as `proxyTo` if it's `null` and then retrievable in the backend configuration?

Reply:
Sorry to push, but could I get another look at this?

Reply:
Right now DB value and config passed in `proxy-server` is the same object. Where it should be present in DB, it should not be passed in proxy-server lib as it is not used there.
Perhaps at this point, we can de-couple these two objects.
I understand the use case and there is no blocker from me 👍

Reply:
Sorry for the delay, I removed `externalUrl` from the `ProxyServer` and added it onto the `ProxyBackend` instead. Does that resolve the issue?